source/dddsample/cargotracker/application/util/SampleDataGenerator.ceylon

```diff
@@ -90,7 +85,7 @@ shared class SampleDataGenerator(
 		// Dropping cargo first won't work since handling events have references
 		// to it.
 		// TODO See if there is a better way to do this.
-		List<Cargo> cargos =CeylonList(entityManager.createQuery("Select c from Cargo c",javaClass<Cargo>())
+		List<Cargo> cargos =CeylonList(entityManager.createQuery("Select c from Cargo c",`Cargo`)
```

Comment:
this is a case where ceylon.interop.persistence can be simplified with [getResults()](https://github.com/ceylon/ceylon-sdk/blob/master/source/ceylon/interop/persistence/Query.ceylon#L67)